pkg/builtin/manifest.yml

```diff
@@ -1258,8 +1258,14 @@ extensions:
   - id: menu
     visualizer: cesium
     type: widget
-    title: Menu
+    title: Menu (legacy)
     description: Menu widgets
+    widgetLayout:
+      floating: true
+      defaultLocation:
+        zone: outer
+        section: left
+        area: top
```

Comment:
```suggestion
```

Floating widgets have no default location. Is it right?

Reply:
Ah that's true! Thank you